packages/models-library/src/models_library/api_schemas_directorv2/clusters.py

```diff
@@ -112,12 +118,14 @@ class ClusterDetailsGet(ClusterDetails):
 
 
 class ClusterCreate(BaseCluster):
-    owner: GroupID | None  # type: ignore[assignment]
+    owner: GroupID | None = None  # type: ignore[assignment]
```

Comment:
change of logic. please revert that one.

Reply:
See: https://github.com/ITISFoundation/osparc-simcore/pull/6333#discussion_r1751389736

Reply:
disagree here. this changes the logic, please check https://docs.pydantic.dev/latest/migration/#unions

Reply:
@sanderegg please, have a look at the documentation of previous Pydantic v1:
https://docs.pydantic.dev/1.10/usage/models/#required-optional-fields